### DISCUSSION TITLE
I can't activate rtsp and webserver

### DISCUSSION THREAD

**Discussion by Marcoske23** [gtxaspec/wz_mini_hacks#462] [Q&A] [answered]
Firmware: 4.36.9.139
Camera: v3 
Steps: 

- I connected the camera to the wifi with the wyze app. 
- Format SD card to FAT32 and copy contents of SD_Root to root of SD card
- I uploaded the files that are in SD_ROOT directly to the SD.
- I modified these values:

> RTSP_LOGIN="admin"
RTSP_PASSWORD="1234"
RTSP_PORT="8554"

>RTSP_HI_RES_ENABLED="True
RTSP_HI_RES_ENABLE_AUDIO="True"
RTSP_HI_RES_FPS=""
RTSP_HI_RES_MAX_BITRATE="2048"
RTSP_HI_RES_TARGET_BITRATE=""
RTSP_HI_RES_ENC_PARAMETER="2"

>RTSP_LOW_RES_ENABLED="false"
RTSP_LOW_RES_ENABLE_AUDIO="false"
RTSP_LOW_RES_FPS=""
RTSP_LOW_RES_MAX_BITRATE=""
RTSP_LOW_RES_TARGET_BITRATE=""
RTSP_LOW_RES_ENC_PARAMETER=""

>RTSP_AUTH_DISABLE="false"
>####WEB####
WEB_SERVER_ENABLED="True"

- From the app I checked the ip: 192.168.50.132
- RTSP LINK i used: rtsp://admin:1234@192.168.50.132:8554/unicast
- WEB SERVER LINK USED: http://192.168.50.132
The video from the camera does not play and I cannot enter the web server although I can connect by ssh.
Any idea how to fix the problem?

**Comment by Bob-AOL** [comment]
The changes need to be saved either via the web tool (at the bottom of the page after editing) or the tool of your choice via SSH. The camera then needs to be rebooted to apply the changes. Top of the web page, Wyze app and command line will all do the trick. I would also suggest setting the target bit rate and fps to 1024 and 20 respectively just to cover all the bases. 

If you happen to be using VLC player and keep getting errors, try it from another device or app such as Tiny Cam or whatever other software you may be using for the purpose. I consistently get inconsistent results with VLC.

If you're convinced wz_mini..conf is configured accordingly but still aren't getting a stream do a port scan to verify 8554 is actually open.

**Reply by Marcoske23** [reply]
I can't access the web server with the IP so I edit from the micro SD.
 I will try to use another application later.
 What program do you recommend to see if the port is open?

**Reply by Bob-AOL** [reply]
I neglected to mention editing the SD card directly -- presumably offline in Windows? If so, Notepad works just fine despite stern warnings to the contrary from various sources. Word Pad won't. The key to using Notepad is _editing and saving the original file_ which was created with the Unix Line Feed convention. Notepad will automatically recognize this and conform accordingly. (If in doubt, you will see (Unix LF) in the bottom panel instead of Windows (CRLF)). If you're editing via SSH to the wz_mini.conf file itself using vi or nano, you obviously need to write out the changes and save accordingly. Either way requires a reboot of the camera for the changes to be applied.

The port scanner of choice for my personal needs happens to be a feature of Wi-Fi Speed Test Analyzer by Analiti Exerts Group, which  is also the best Wi-Fi analyzer and home network tool out there today. Nothing else even comes close. It's also free in it's basic form and only $1.99/yr. to get rid of the ads. Designed for Android devices including FireTV and Android TV platforms.  analiti.com Just get it and don't look back.

This scan is a v3 with the Wyze RTSP Beta firmware using the wz_mini USB/ETH feature with a UCTRONICS adapter (Realtek). (A wz_mini scan would show port 8554 for RTSP if set as such in the wz_mini.conf file.)

![Analiti Screenshot_resized](https://user-images.githubusercontent.com/110629141/212957179-a83423ab-f302-4c86-a630-cc0aa23c234d.png)

**Comment by dsmath** [comment, marked as answer]
double check your changes to wz_mini.conf

note, you pasted in upper case "T" for the "True" changes you made. 

pretty sure settings all needs to be lower case; linux is finicky that way.

also see a missing double quote (") on one of your settings.

**Reply by Marcoske23** [reply]
> double check your changes to wz_mini.conf
> 
> note, you pasted in upper case "T" for the "True" changes you made.
> 
> pretty sure settings all needs to be lower case; linux is finicky that way.
> 
> also see a missing double quote (") on one of your settings.

Today I was checking the configuration and that's it, I put "True" and it should be lowercase. In the same way I realized that the firmware of my camera was a strange number so I updated it and it already worked. Thank you very much for your help!
